chunkloader.cpp

```diff
@@ -44,7 +46,7 @@ void ChunkLoader::run() {
     return;
   }
   // get existing Chunk entry from Cache
-  Chunk *chunk = cache.fetchCached(cx, cz);
+  QSharedPointer<Chunk> chunk(cache.fetchCached(cx, cz));
```

Comment:
How about using `auto` more?
`auto chunk = cache.fetchCached(cx, cz));`
Doesn't matter then if it's a `QSharedPointer`, `std::shared_ptr`, naked pointer or whatever else.

Reply:
I don't like auto that much, as you do not see the type of a variable in a normal editor.
(I know that this answer is contradictory to your (good) argument. Probably it is just personal style how much auto you use.)